charts/vela-core/templates/addons-default/terraform.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Initializer
+metadata:
+  annotations:
+    addons.oam.dev/description: Terraform Controller is a Kubernetes Controller for
+      Terraform.
+    helm.sh/hook: post-install, post-upgrade, pre-delete
+  name: terraform
+  namespace: {{.Values.systemDefinitionNamespace}}
+spec:
+  appTemplate:
+    spec:
+      components:
+      - name: terraform-controller
+        properties:
+          chart: terraform-controller
+          repoType: helm
+          url: https://charts.kubevela.net/addons
+          version: 0.1.19
+        type: helm
+      - name: terraform-system
+        properties:
+          apiVersion: v1
+          kind: Namespace
+          metadata:
+            name: terraform-system
+        type: raw
+    status: {}
+  dependsOn:
+  - ref:
+      apiVersion: core.oam.dev/v1beta1
+      kind: Initializer
+      name: fluxcd
+      namespace: vela-system
+status:
+  observedGeneration: 0
```

Comment:
don't add this file in this PR, not related

Reply:
done